app/controllers/App.scala

```diff
@@ -40,4 +50,21 @@ class App(val wsClient: WSClient, val config: Config, val db: MetricsDB) extends
         case Left(_) => InternalServerError("Not able to parse json.")
       })
   }
+
+  def saveMetric() = CORSable(config.workflowUrl) {
+    APIHMACAuthAction { req =>
+      req.body.asJson.map(_.toString) match {
+        case Some(metricOptString) =>
+          val result = for {
+            metricOptJson <- stringToJson(metricOptString)
+            composerId <- metricOptJson.hcursor.downField("composerId").as[String].fold(processException, cId => Right(cId))
+            metric <- db.updateOrInsert(db.getPublishingMetricsWithComposerId(Some(composerId)), metricOptJson)
+          } yield metric
+          result.fold(
+            err => InternalServerError(s"Something bad happened while trying to post a metric: ${err.message}"),
```

Comment:
It would probably be good to log this as well so logging from other services doesn't fail silently

Reply:
Good point, I'll log it.